fml/trace_event.cc

```diff
@@ -22,6 +23,21 @@ namespace tracing {
 
 #if TIMELINE_ENABLED
 
+static AsciiTrie s_whitelist;
```

Comment:
Nit on name: go/fluttercpp#Constant_Names (the final paragraph on static storage duration).

Reply:
done, followed convention from the file with "g" prefix.